pkg/local/vmis.go

```diff
@@ -119,6 +119,7 @@ func createInstance(binding *types.SyntheticBinding, verrazzanoURI string, enabl
 			AutoSecret:      true,
 			SecretsName:     constants.VmiSecretName,
 			CascadingDelete: true,
+			API:             vmov1.API{Replicas: 0},
```

Comment:
I'm wondering if this change is necessary, if we allow the default of 0 to apply.  Then at some point down the road we can enable it if we decide we want it.

Reply:
Closing this PR